### PR TITLE
Document how to make a release

### PR DESCRIPTION
**Pull request by erbridge**

### CODE REVIEW

File: RELEASING.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+Go through each one and test any related tickets are truly fixed on the staging site and the new behaviour works as described. If there are any issues, make a call as to whether they need resolving before the release can happen or not. If they're not blocking issues, reopen the tickets or open new tickets for them instead and continue with the release.
+
+## 2. Open a pull request
+
+If everything looks good to publish, open a pull request from `main` into `production` and give it a title reflecting the new version we're releasing.
+
+## 3. Edit the release
+
+Edit the draft release to reflect the reality of what we're releasing after reviewing for any unresolved issues. The main aim here is to avoid misleading our future selves with "Fixed X" items when we actually didn't. **Importantly, don't remove any PR links.** Even if the thing they related to didn't work, we want the reference there to know what's included.
```
[the remark this answers is not in the document]
Comment (erbridge) [reply RELEASING.md:15]: I think you're confusing pull requests from GitHub Releases here. This is in reference to the link in step 1's heading.

File: RELEASING.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+Set the release up to create a new tag of the head of the `main` branch on publish and set the release title to match.
+
+## 4. Merge the pull request
+
+Do a **merge commit based merge**. Do not squash or rebase! And wait for CI!
+
+## 5. Publish the release
+
+Press the "publish" button on the release.
```
[the remark this answers is not in the document]
Comment (erbridge) [reply RELEASING.md:25]: It's on the GitHub Releases release. I take your point, but I'm also not very interested in spending a lot of effort on some documentation mostly for myself. Unless you think this is actually wrong (the wrong process or incorrect instructions), do you think we could merge it and you (or anyone else) can offer improvements as needed?

File: RELEASING.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (ivan-kocienski-gfsc) [reply RELEASING.md:15]: ah okay

**Review by ivan-kocienski-gfsc — APPROVED** [review]
This looks good to me

**Pull request merged**